tests/integration/test_keeper_zookeeper_converter/test.py

```diff
@@ -24,6 +24,8 @@ def start_zookeeper():
 
 def stop_zookeeper():
     node.exec_in_container(["bash", "-c", "/opt/zookeeper/bin/zkServer.sh stop"])
+    while node.get_process_pid("zookeeper") != None:
```

Comment:
Can you add global timeout like 60 seconds to stop so? Otherwise it will fail with 900 seconds test timeout only. And we can add better failure status that zk did not complete snapshot in X /"seconds